components/kyma-environment-broker/cmd/deprovisionretrigger/main.go

```diff
@@ -0,0 +1,145 @@
+package main
+
+import (
+	"context"
+	"fmt"
+	"os"
+	"time"
+
+	"github.com/kyma-project/control-plane/components/kyma-environment-broker/internal"
+
+	"github.com/kyma-project/control-plane/components/kyma-environment-broker/internal/broker"
+	"github.com/kyma-project/control-plane/components/kyma-environment-broker/internal/events"
+	"github.com/kyma-project/control-plane/components/kyma-environment-broker/internal/storage"
+	"github.com/kyma-project/control-plane/components/kyma-environment-broker/internal/storage/dbmodel"
+	"github.com/kyma-project/control-plane/components/schema-migrator/cleaner"
+	log "github.com/sirupsen/logrus"
+	"github.com/vrischmann/envconfig"
+)
+
+type BrokerClient interface {
+	Deprovision(instance internal.Instance) (string, error)
+}
+
+type Config struct {
+	Database storage.Config
+	Broker   broker.ClientConfig
+	DryRun   bool `envconfig:"default=true"`
+}
+
+type DeprovisionRetriggerService struct {
+	cfg             Config
+	filter          dbmodel.InstanceFilter
+	instanceStorage storage.Instances
+	brokerClient    BrokerClient
+}
+
+func main() {
+	time.Sleep(20 * time.Second)
```

Comment:
nit: not a big fan of 20s sleep, do we know what it needs to wait for and can we be more deterministic at finding out the pre-requisite to be ready?

Reply:
@wozniakjan We need to ask @ralikio - this is "skeleton" part added during migrator migration.

Reply:
I am not a fan of Sleep either. I am fan of sleep.

Reply:
This is not mine!

Reply:
Removed.

Reply:
Hello! Not originally mine. Maybe I am an author because of refactoring. Nevertheless, I am all for removal!